README.md

```diff
@@ -6,6 +6,13 @@ bemenu
 
 Dynamic menu library and client program inspired by dmenu
 
+## Usage
+For more info see `bemenu-run -h`, e.g. if you want to show it at the bottom of the screen add
+```
+set $menu bemenu-run -b
+```
+to your sway config file.
```

Comment:
This project isn't really related to sway so this is bit weird.

Reply:
ah, ok...